spiped/main.c

```diff
@@ -274,50 +285,69 @@ main(int argc, char * argv[])
 			warnp_syslog(1);
 	}
 
+	/* Resolve bind address. */
```

Comment:
If this section occurred after resolving `sas_t`, then we'd have fewer modified lines, which always makes it easier to read & check commits.

Reply:
Come to think of it... if we have the modified `sock_resolve_one()`, then the `errX` labels would only increase by 1 instead of 2 for this code block.  In general, I think that's a good thing.

However, it might be annoying for you to modify all this code twice.  So maybe hold off on this change until the libcperciva infrastructure is finished.